src/components/LayerUI.tsx

```diff
@@ -660,7 +669,15 @@ const LayerUI = ({
         />
       )}
       {appState.showHelpDialog && (
-        <HelpDialog onClose={() => setAppState({ showHelpDialog: false })} />
+        <HelpDialog
+          onClose={() => {
+            const helpIcon = document.querySelector(
+              ".help-icon",
+            )! as HTMLElement;
```

Comment:
this is too generic — can break at any time (+ isn't multi-component friendly)

Reply:
this will be fixed when we start storing active element as discussed in separate PR and right now we are doing this at other places as well example collab button for manually focussing.